lib/block.js

```diff
@@ -7,35 +7,39 @@
  */
 
 const TwoPowTwenty = 1048575
+
 /**
  *
- * @param {Number} sequenceNumber The block sequence number.
- * @param {Boolean} moreBlocks If there are more blocks to follow
- * @param {Number} blockSize
+ * @param numOrBlockState The block sequence number or a block state object.
+ * @param more Can indicate if more blocks are to follow.
```

Comment:
"if" somehow indicated that it is a boolean ... but in fact it is a nunber?

Reply:
and if number with only 0/1 why not boolean?

Reply:
I think I changed it to a boolean first (there was some inconsistency as the block1 implementation used numbers, if I recall correctly) but then used 0/1 instead as the `more` flag can have different meanings depending on the context (see https://datatracker.ietf.org/doc/html/rfc7959#section-2.2). Therefore the "Can indicate". But I guess we could also use boolean values here.

Reply:
Ok if specs have it as number we better stay near to the specs so maybe just change the text maybe to indicate that